.github/ISSUE_TEMPLATE/large-dataset-datacap-application.yaml

```diff
@@ -344,6 +343,27 @@ body:
       - More Than 15PiB
   validations:
     required: true
+- type: input
+  attributes:
+    label: What is the size of your dataset (One Copy)
```

Comment:
Looks like other prompts are not questions (such as Number of replicas). Should we reword this to a statement, such as: "Expected size of single dataset (one copy)"